### ISSUE TITLE
On mobile save/copy is immediately triggered

### ISSUE THREAD

**Issue by jalezi** [subhoghoshX/laureate#29]
### Description

Secondary event is triggered immediately after "Export" button is clicked.
For a moment border is not rounded, especially when saving image while it takes a little bit longer than coping.

### Scenario

Tested on iPhone SE.

When "Export" button is clicked either "save" or "copy" event is triggered. You have to hit exact point not to trigger any event except that "Export" button is expanded.

### Screenshots
![IMG_C5E80309B90D-1 2](https://user-images.githubusercontent.com/44704999/193535238-a91ce447-a0f2-415d-ac29-d18d2b1ea372.jpeg)

### Video

https://user-images.githubusercontent.com/44704999/193538449-f1963122-be5f-4872-86e0-236c7cde6d27.mov

### Tasks

- [ ] save/copy not triggered immediately
- [ ] rounded buttons 

#### save/copy not triggered immediately

??? have no idea

#### rounded buttons

I guess for border-radius adding "rounded-full" class to both buttons would do the trick. I'm Tailwind noob.

### Additional enhancement

I believe user should be notified what is going on. E.g. notification when image is copied to clipboard.

**Comment by Karthik-315** [comment]
@subhoghoshX, does this issue still exist, if yes I have an idea that might work. I am thinking of playing around with the `scale` property of the **Save** and **Copy** buttons when tapping the **Export** button. 

Let me know if it works for you. I can work on that.

**Comment by subhoghoshX** [comment]
Yes, it still exists, but only the first one i.e. the event triggered immediately. I don't see the square corner issue, not sure which browser the OP used and if it's up to date.

Sure feel free to play with it.

**Comment by jalezi** [comment]
> Yes, it still exists, but only the first one i.e. the event triggered immediately. I don't see the square corner issue, not sure which browser the OP used and if it's up to date.
> 
> Sure feel free to play with it.

iPhone SE and iOS latest.

[Picture and video from chrome for iOS ⬆️](https://github.com/subhoghoshX/laureate/issues/29#issue-1394387330)

safari ⬇️ 
![IMG_C514ECCC068D-1](https://user-images.githubusercontent.com/44704999/195996078-12271eec-4e43-44c7-b16c-81092c2a0d5a.jpeg)

**Comment by subhoghoshX** [comment]
Interesting gotta look at it. Thanks.

**Comment by Karthik-315** [comment]
Thanks, I'll work on this issue and update you soon.
